http/jsonapi/generator/internal/fueling/open-api_test.go

```diff
@@ -536,17 +539,25 @@ type Service interface {
 }
 
 /*
-Router implements: PACE Fueling API
+RouterWithAuthentication implements: PACE Fueling API
 
 Fueling API
 */
-func Router(service Service) *mux.Router {
+func RouterWithAuthentication(service Service, authBackend AuthenticationBackend) *mux.Router {
+	if authBackend != nil {
+		authBackend.Init()
+	}
 	router := mux.NewRouter()
 	// Subrouter s1 - Path: /fueling
 	s1 := router.PathPrefix("/fueling").Subrouter()
-	s1.Methods("GET").Path("/beta/gas-stations/{gasStationId}/pumps/{pumpId}/wait-for-status-change").Handler(WaitOnPumpStatusChangeHandler(service)).Name("WaitOnPumpStatusChange")
-	s1.Methods("GET").Path("/beta/gas-stations/{gasStationId}/pumps/{pumpId}").Handler(GetPumpHandler(service)).Name("GetPump")
-	s1.Methods("POST").Path("/beta/gas-station/{gasStationId}/payment").Handler(ProcessPaymentHandler(service)).Name("ProcessPayment")
-	s1.Methods("POST").Path("/beta/gas-stations/{gasStationId}/approaching").Handler(ApproachingAtTheForecourtHandler(service)).Name("ApproachingAtTheForecourt")
+	s1.Methods("GET").Path("/beta/gas-stations/{gasStationId}/pumps/{pumpId}/wait-for-status-change").Handler(WaitOnPumpStatusChangeHandler(service, authBackend)).Name("WaitOnPumpStatusChange")
+	s1.Methods("GET").Path("/beta/gas-stations/{gasStationId}/pumps/{pumpId}").Handler(GetPumpHandler(service, authBackend)).Name("GetPump")
+	s1.Methods("POST").Path("/beta/gas-station/{gasStationId}/payment").Handler(ProcessPaymentHandler(service, authBackend)).Name("ProcessPayment")
+	s1.Methods("POST").Path("/beta/gas-stations/{gasStationId}/approaching").Handler(ApproachingAtTheForecourtHandler(service, authBackend)).Name("ApproachingAtTheForecourt")
 	return router
 }
+
+// Router kept for backward compatibility. Please use RouteWithAuthentication
+func Router(service Service) *mux.Router {
+	return RouterWithAuthentication(service, nil)
+}
```

Comment:
Just leaving a note here that, unlike what I previous said, we should keep this `Router` method.